webapp/first_snap/content/pre-built/package.yaml

```diff
@@ -0,0 +1,60 @@
+name: test-geekbench4-{name}
+download: git clone https://github.com/snapcraft-docs/geekbench4
+createDir: |
+  cd geekbench4
+  $EDITOR snapcraft.yaml
+metadata: |
+  name: geekbench4
+  version: 4.2.0
+  summary: Cross-Platform Benchmark
+  description: |
+    Geekbench 4 measures your system's power and tells you whether your
```

Comment:
The line breaks here need adjusting as it causes bad wrapping on the site:
![screenshot_2018-12-04 snapcraft - snaps are universal linux packages](https://user-images.githubusercontent.com/479384/49434244-a97fe680-f7ab-11e8-9183-6de46e410dac.png)

Reply:
Fixed as 92a4c30.